using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -170,34 +190,54 @@ ACPCampaign.resetLinkageFields();
 
 ## Set up push messaging
 
+To enable push messaging with Adobe Campaign, the push identifier that is received from the Apple Push Notification Service (APNS) or Firebase Cloud Messaging (FCM) must be sent to the Adobe Identity service by calling `setPushIdentifer`. After the API is invoked, a network request is made to Campaign that contains the message interaction event. For more information about Campaign message interaction events, see [Implementing local notification tracking](https://helpx.adobe.com/campaign/kb/local-notification-tracking.html#Description).
+
+For more information about setting up your iOS app to connect to APNS and retrieve a device token that will be used as a push identifier, see https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns?language=objc.
```

Comment:
- Please update the URL as follows:
[Registering Your App with APNs](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns?language=objc).